compiler/il/OMRILOps.hpp

```diff
@@ -1429,6 +1433,12 @@ class ILOpCode
          case TR::fneg:
          case TR::dneg:
             return TR::vneg;
+
+         case TR::iabs: return ILOpCode::createVectorOpCode(OMR::vabs, TR::DataType::createVectorType(TR::Int32,  vectorLength)).getOpCodeValue();
+         case TR::labs: return ILOpCode::createVectorOpCode(OMR::vabs, TR::DataType::createVectorType(TR::Int64,  vectorLength)).getOpCodeValue();
+         case TR::fabs: return ILOpCode::createVectorOpCode(OMR::vabs, TR::DataType::createVectorType(TR::Float,  vectorLength)).getOpCodeValue();
+         case TR::dabs: return ILOpCode::createVectorOpCode(OMR::vabs, TR::DataType::createVectorType(TR::Double, vectorLength)).getOpCodeValue();
```

Comment:
I believe these can all be collapsed into a single `ILOpCode::createVectorOpCode` just like on L1393.